compose/redis.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.4'
+services:
+  ccd-redis:
+    image: "redis:alpine"
+    container_name: ccd-redis
+    command: redis-server
+    ports:
+      - 6379:6379
+    volumes:
+      - redis-data:/data
+      - redis-conf:/usr/local/etc/redis/redis.conf
+volumes:
```

Comment:
we could leave default volumes .

Reply:
I think leaving the default is fine (without volumes), because we have no pre-setup required, Data Store does all the configuration it needs on startup 👍

Reply:
DONE